### ISSUE TITLE
Build documentation hopelessly outdated on master

### ISSUE THREAD

**Issue by mpreisler** [OpenSCAP/openscap#1069]
#### Description of Problem:
Master is built using cmake but there are tons of mentions of autogen, automake, etc... still in master. All the docs need to be updated.

Furthermore, Windows is considered second citizen platform (there is a special chapter for it and it says "openscap can also be built on windows...") which is not the case any-more. The README assumes you are using Linux and has no mention of Windows or OSX. We need to restructure the docs to support Linux, MacOS X and Windows builds equally.

I am mainly talking about `docs/manual/manual.adoc` and README here but please... `grep` the whole repo when fixing this.

#### OpenSCAP Version:
```
git show
commit 0f648dc2d6efe90b9519a333dd25de697c008f1d (HEAD -> python2_disabled_if_not_found, upstream/master, master)
Merge: 5fdc2ad78 9239c06b5
Author: Watson Yuuma Sato <wsato@redhat.com>
Date:   Fri Jun 15 13:26:29 2018 +0200

    Merge pull request #1050 from cipherboy/rm-oval-probe-dir

    Remove occurrences of OVAL_PROBE_DIR in master
```

#### Operating System & Version:
Vokna 10

**Comment by mpreisler** [comment]
I agree with your approach. I'd maybe suggest that users might be interested in how to compile as well. Especially on Linux platforms users are often expected to compile their packages.

**Comment by jan-cerny** [comment]
@mpreisler Where do you think would be the best place for this information then?

**Comment by jan-cerny** [comment]
@mpreisler Could it be something like #1080 ?

**Comment by matejak** [comment]
Closed by #1080
